praw/models/reddit/live.py

```diff
@@ -217,6 +128,141 @@ def remove_invite(self, redditor):
         self.thread._reddit.post(url, data=data)
 
 
+class LiveThread(RedditBase):
+    """An individual LiveThread object."""
+
+    STR_FIELD = 'id'
+
+    @property
+    def contrib(self):
```

Comment:
Since `mod` is used for moderation, `contribution` sounds less consistent for me.